src/openforms/prefill/contrib/stufbg/tests/test_co_sign.py

```diff
@@ -36,8 +34,15 @@ def test_store_names_on_co_sign_auth(self):
                 "fields": {},
             }
         )
+        return_value = loader.render_to_string(
+            "stuf_bg/tests/responses/StufBgResponse.xml"
+        )
```

Comment:
I dislike this low-level stuff that's irrelevant to the test itself - what I care about is that `StufBgResponse.xml` is used for the mock. Why does the mocker approach not work?